src/bin/wd_copier.rs

```diff
@@ -12,10 +15,40 @@ struct Opt {
     dst: PathBuf,
 }
 
+fn get_dirty_bytes() -> u64 {
+    let meminfo = procfs::Meminfo::new().unwrap();
```

Comment:
This error should be handled instead of unwrapping

Reply:
Handled in b63e642. Additionally in the error case it now skips the progress bar altogether and falls back to the old `println!`